docs/communicate-with-javascript.md

```diff
@@ -1146,12 +1163,14 @@ var value = [
 ];
 ```
 
-#### Using `Js.t` objects
+#### “Objects with extensible shape” or unknown shape?
 
 Alternatively to records, Melange offers another type that can be used to
 produce JavaScript objects. This type is `'a Js.t`, where `'a` is an [OCaml
 object](https://v2.ocaml.org/manual/objectexamples.html).
 
+<!-- I would explain the difference here, since it's a source of confusion for some users -->
```

Comment:
There are two paragraphs just below that mention the differences:

- The first one mentions "no type declaration is needed in advance"
- The second one mentions "an object type can be coerced to another with fewer values or methods"

What other differences do you think we should mention, or in what order?